apps/web/pages/sandbox/List.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from "react";
 
```

Comment:
Do we still use these sandbox pages actually? Otherwise I'd be more than happy to delete them 😁

Reply:
I would say go for it. The less code to maintain the better

Reply:
agree